docs/guides/aws-private-link-workspace.md

```diff
@@ -0,0 +1,261 @@
+---
+page_title: "Enable Backend AWS PrivateLink for Databricks Workspace"
+---
+
+# Deploying pre-requisite resources and enabling PrivateLink connections (AWS Preview)
+
+-> **Private Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+
+Databricks PrivateLink support enables private connectivity between users and their Databricks workspaces and between clusters on the data plane and core services on the control plane within the Databricks workspace infrastructure. You can use Terraform to deploy the underlying cloud resources and the private access settings resources automatically, using a programmatic approach. This guide assumes you are deploying into an existing VPC and you have set up credentials and storage configurations as per prior examples, notably here.
+
+![Private link backend](../../docs/images/aws-e2-private-link-backend.png)
+
+This guide uses the following variables in configurations:
+
+- `databricks_account_username`: The username an account-level admin uses to log in to  [https://accounts.cloud.databricks.com](https://accounts.cloud.databricks.com).
+- `databricks_account_password`: The password for `databricks_account_username`.
+- `databricks_account_id`: The numeric ID for your Databricks account. When you are logged in, it appears in the bottom left corner of the page.
+- `vpc_id` - The ID for the AWS VPC 
+- `network_name` - Name for your Databricks-configured network
+- `region` - AWS region
+- `existing_network_sg` - Security groups set up for the existing VPC
+- `existing_network_subnets` - Existing subnets being used for the customer managed VPC
+- `workspace_vpce_service` - Choose the region-specific service endpoint from this table.
+- `relay_vpce_service` - Choose the region-specific service from this table.
+- `vpc_cidr_block` - CIDR range for the VPC being deployed into
```

Comment:
you already pass `vpc_id` in the argument, so you can get it from a data resource. remove it from variables.